### PR TITLE
Add log book tab to Journal, close #34

### LINKED ISSUE #34: Feature suggestion: add a game log in the "journal" GUI
Currently, the game log is visible in the bottom left corner, but once the messages disappear from it, there is no way to see them.

I suggest to add another tab to the "Journal" GUI, so the older messages can be read from it. 

If we decide to follow through with it, I might try to implement this, but since it's quite complicated at first glance, I might not be able to do it.

### PR DESCRIPTION
**Pull request by swooboo**
In this PR:

The Journal window features two tabs - levels and log book, the levels tab features the levels, like the journal did before, and the log book shows last 30 messages.

Please review this carefully before merging. I'm still not 100% familiar with the code and might have broken something. So if something is missing / broken, let me know.

### CODE REVIEW

File: PixelDungeon/src/main/java/com/watabou/pixeldungeon/windows/WndJournal.java [hunk 1]
```diff
@@ -30,14 +30,25 @@
...
-	private static final int ITEM_HEIGHT	= 18;
-	
 	private static final String TXT_TITLE	= Game.getVar(R.string.WndJournal_Title);
+	private static final String TXT_LEVELS	= Game.getVar(R.string.WndJournal_Levels);
+	private static final String TXT_LOGBOOK	= Game.getVar(R.string.WndJournal_Logbook);
+
+	private Text       txtTitle;
+	private ScrollPane list;
+
+	private static boolean showLevels = true;	// Indicates which tab is visible
```
Comment (Mikhael-Danilov) [inline PixelDungeon/src/main/java/com/watabou/pixeldungeon/windows/WndJournal.java]: Can we do not use this field, but instead move tab specific functionality to respective tab class, and then just pass required calls to WndTabbed.selected ?

File: PixelDungeon/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -184,6 +184,8 @@
 
     <!-- STRINGS OF WndJournal CLASS -->
     <string name="WndJournal_Title">Journal</string>
+    <string name="WndJournal_Levels">Levels</string>
```
Comment (Mikhael-Danilov) [inline PixelDungeon/src/main/res/values/strings.xml]: Can you please add all new strings to strings_remix_next.xml?
This is sound a bit strange I know, but it let me feel safe while uploading new file to transifex.

File: PixelDungeon/src/main/java/com/watabou/pixeldungeon/windows/WndJournal.java (hunk 1 above)
Comment (swooboo) [reply PixelDungeon/src/main/java/com/watabou/pixeldungeon/windows/WndJournal.java]: Hm this sounds good. I tried to mimic the WndCatalogus behavior, but I had a feeling that it's not the best way. I'll work on this.

File: PixelDungeon/src/main/res/values/strings.xml (hunk 2 above)
Comment (swooboo) [reply PixelDungeon/src/main/res/values/strings.xml]: Should I move them or should they be in both files?

Comment (Mikhael-Danilov) [reply PixelDungeon/src/main/res/values/strings.xml]: Please move them.

Comment (swooboo) [reply PixelDungeon/src/main/res/values/strings.xml]: Done. Moved to the `strings_remix_next.xml`, but only the 2 new strings I added.

File: PixelDungeon/src/main/java/com/watabou/pixeldungeon/windows/WndJournal.java (hunk 1 above)
Comment (swooboo) [reply PixelDungeon/src/main/java/com/watabou/pixeldungeon/windows/WndJournal.java]: @Mikhael-Danilov, I need clarification. Do you want to remove the `showLevels` flag from `WndJournal.java`, but leave the `ListItem` class unchanged? Or are you suggesting 2 separate classes, like `ListLevelItem` and `ListLogItem`? Thanks.

Comment (Mikhael-Danilov) [reply PixelDungeon/src/main/java/com/watabou/pixeldungeon/windows/WndJournal.java]: 2 separate classes of course.

**Comment by swooboo** [comment]
Thanks @Mikhael-Danilov, another question. Should I debase to master before you merge? That way the merge will be fast-forward, even if not, the only change that will happen is my code, so there will be no conflicts etc.

**Comment by Mikhael-Danilov** [comment]
Thanks to you! Yes, rebase-to-master before merge is preferred way.

**Pull request merged**